src/icon/PaginationFirstIcon.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import { SvgIcon, ISvgIconProps } from "../util/react/SvgIcon";
+
+export interface IPaginationFirstIconProps extends ISvgIconProps {
+
+}
+
+export class PaginationFirstIcon extends React.Component<IPaginationFirstIconProps> {
+    render() {
+        return (
+            <SvgIcon {...this.props}>
+                <g stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
+                    <g transform="translate(-436.000000, -6.000000)" fill="currentColor" fillRule="nonzero">
+                        <g transform="translate(304.000000, 0.000000)">
```

Comment:
Minor optimization: this is a common "glitch" when exporting from sketch. If you add up the translate offsets from all of the group elements, the sum is 0, so in this case the only effect is produced by the last group that does +12, rotate, -12.
```
<g transform="translate(12, 12) rotate(180) translate(-12, -12)" fill="currentColor" fillRule="nonzero">
```